src/router/Router.ts

```diff
@@ -2,6 +2,7 @@ import { createVNode } from 'inferno';
 import Component from 'inferno-component';
 import match, { matchPath } from './match';
 import RouterContext from './RouterContext';
+import VNodeFlags from 'inferno-vnode-flags';
```

Comment:
stupid question: Why
import VNodeFlags from 'inferno-vnode-flags';
 is needed in these files?